packages/help-center/src/components/help-center-sibyl-articles.tsx

```diff
@@ -114,9 +114,7 @@ export function SibylArticles( {
 			const hasPostId = 'post_id' in article && article.post_id;
 			return {
 				...article,
-				url: hasPostId
-					? getPostUrl( article as Article, message, articleCanNavigateBack )
-					: article.link,
+				url: getPostUrl( article as Article, message, articleCanNavigateBack ) ?? article.link,
```

Comment:
Yes, it is relevant. The `url` can no longer be `undefined` on the `ConfigureableLink` component. The `ConfigureableLink` component extends its props with `LinkProps` from `react-router-dom`. 

Also, if you were to study the return value of the conditional assignment, it will not resolve to `undefined`. The `getPostUrl` function, checks whether the article prop has an ID.  It will return `undefined` if the article does not have an ID. We already ensure that the article passed in has an ID with the `hasPostId` const. However, typescript is not aware of this, and hence it believes the return type can still be `undefined` when an article has an ID.

![Screen Shot 2023-04-05 at 15 34 57](https://user-images.githubusercontent.com/2019970/230096906-11858c78-d158-4dd5-b799-6c4644912abe.png)